BAZProject/Extensions/StringExtension.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  StringExtension.swift
+//  BAZProject
+//
+//  Created by Luis Alberto Perez Villar on 02/02/23.
+//
+
+import Foundation
+
+extension String {
+    
+    /**
+     Convierte el valor de la cadena del formato inicial al nuevo, si el valor de la cadena no corresponde a una fecha regresa un nulo
+     
+     - Parameters:
+        - format: corresponde al formato actual de la fecha
+        - newFormat: corresponde al nuevo formato al que desea convertir la fecha
+     - Returns: regresa una cadena con la fecha en el formato recibido, de no ser posible regresa un valor nulo
+     
+     ````
+     publishedDate.convertDate(format: "yyyy-MM-dd", with: "dd MMMM yyyy")
+     ````
+    */
+    func convertDate(format: String, with newFormat: String) -> String? {
```

Comment:
Me suena un poco mejor está definición de este método.
`func convertDate(from format: String, to newFormat: String)`

Reply:
No me hace sentido que cambies una fecha que se encuentra en un `String` y devuelvas otro `String`. ¿Es por que lo quieres presentar en otro formato en un label?